### PR TITLE
Update index.css

### PR DESCRIPTION
**Pull request by Eylas**
Updated responsiveness for landing page service cards to include 1400/1100/700 px widths

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -461,9 +462,30 @@ nav ul li {
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+  }
 }
 
 @media only screen and (max-width: 750px) {
+  .services {
+    height: auto;
```
Comment (Spacebjorn) [inline css/index.css:486]: Height is already set to to auto in max-width: 1400. So it's already covered. Removed this segment

File: css/index.css [hunk 2]
```diff
@@ -498,13 +520,15 @@ nav ul li {
 @media screen and (max-width: 700px) {
   .services {
     padding-bottom: 25px;
+    height: auto;
```
Comment (Spacebjorn) [inline css/index.css:523]: same thing as above

File: css/index.css [hunk 3]
```diff
@@ -498,13 +520,15 @@ nav ul li {
 @media screen and (max-width: 700px) {
```
Comment (Spacebjorn) [inline css/index.css:520]: should probably be part of the "max-width: 750px" section instead of its own meda query

**Review by Spacebjorn — CHANGES_REQUESTED** [review]

**Comment by Spacebjorn** [comment]
Close #88

File: css/index.css (hunk 1 above)
Comment (Eylas) [reply css/index.css:486]: Resolved

File: css/index.css (hunk 2 above)
Comment (Eylas) [reply css/index.css:523]: Resolved

**Review by Spacebjorn — APPROVED** [review]

**Pull request merged**